packages/lesswrong/server/vulcan-core/start.ts

```diff
@@ -1,7 +1,9 @@
 import { SyncedCron } from 'meteor/littledata:synced-cron';
 import { Meteor } from 'meteor/meteor';
 import { Inject } from 'meteor/meteorhacks:inject-initial';
-import { mailUrlSetting } from '../../lib/instanceSettings';
+import { DatabaseServerSetting } from '../databaseSettings';
+
+export const mailUrlSetting = new DatabaseServerSetting<string | null>('mailUrl', null) // The SMTP URL used to send out email
```

Comment:
I'm not sure what this is doing, seems good for someone to doublecheck this is right before merging to master.